### DISCUSSION TITLE
Cannot create v10.0 projects

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#554] [Doodba - The image] [answered]
Some time ago I create an v10 project using Doodba and everything works fine. Today i need to install one extra lib in this project image and after modify the dependencies file runs `invoke img-build`. The result is failure because this base image is build over a debian jessie and this version of debian is discontinued and repositories has been changed to archive.
```
Err http://deb.debian.org jessie/main amd64 Packages
Err http://deb.debian.org jessie/main amd64 Packages
Err http://deb.debian.org jessie/main amd64 Packages
```
I know that v10.0 is an old version but some clients still use it.  I assume that v7 is in the same situation.
What can be done to overcome those errors and rebuild and use doodba for this old versions?

**Comment by sergio-teruel** [comment]
Hi! This occours because the main debian jessie repos has been removed.
To solve this (v10 is deprecated so this is weird fix), do:
1 - Create a file under /odoo/custom/build.d like 05-update-debian-sources
2 - edit the file with this content

`#!/bin/bash
#First remove sources
sed -i '/deb http:\/\/deb.debian.org\/debian jessie main/d' /etc/apt/sources.list
sed -i '/deb http:\/\/security.debian.org\/debian-security jessie\/updates main/d' /etc/apt/sources.list

#Second add archive sources
echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie main" > /etc/apt/sources.list.d/jessie.list
echo "deb [check-valid-until=no] http://archive.debian.org/debian-security jessie/updates main" >> /etc/apt/sources.list.d/jessie.list

#PostgreSql repositories
#First one removes the current line
echo "deb https://apt-archive.postgresql.org/pub/repos/apt jessie-pgdg-archive main" > /etc/apt/sources.list.d/postgresql.list

#Second one appends
echo "deb-src https://apt-archive.postgresql.org/pub/repos/apt jessie-pgdg-archive main" >> /etc/apt/sources.list.d/postgresql.list

apt-get -o Acquire::Check-Valid-Until=false update`

3 - Build your project

**Reply by rquevedo** [reply, marked as answer]
Hi @sergio-teruel, I really appreciate your response. I have to do some changes to your idea, but the spirit is the same. Finally, the file was like this:

```
#!/bin/bash
#First remove sources
sed -i 's+deb http://deb.debian.org/debian jessie main++g' /etc/apt/sources.list
sed -i 's+deb http://security.debian.org/debian-security jessie/updates main++g' /etc/apt/sources.list

#Second add archive sources
curl -k --output freexian-archive-keyring_2022.06.08_all.deb https://deb.freexian.com/extended-lts/pool/main/f/freexian-archive-keyring/freexian-archive-keyring_2022.06.08_all.deb && dpkg -i freexian-archive-keyring_2022.06.08_all.deb
echo "deb http://deb.freexian.com/extended-lts jessie main contrib non-free" > /etc/apt/sources.list.d/jessie.list

```
Thank you very much
